NDTensors/src/backup/arraystorage/blocksparsearray/storage/unwrap.jl

```diff
@@ -1 +1,5 @@
-Unwrap.parenttype(::Type{<:BlockSparseArray{<:Any,<:Any,P}}) where {P} = P
+function TypeParameterAccessors.position(
+  ::Type{<:BlockSparseArray}, ::typeof(TypeParameterAccessors.parenttype)
+)
+  return Position(3)
+end
```

Comment:
```suggestion
using TypeParameterAccessors: parenttype

function TypeParameterAccessors.position(
  ::Type{<:BlockSparseArray}, ::typeof(parenttype)
)
  return Position(3)
end
```

Reply:
Also maybe add an inverse definition `TypeParameterAccessors.parameter_function`.

Reply:
@kmp5VT is this resolved?